packages/airnode-validator/src/config/config.ts

```diff
@@ -15,14 +16,24 @@ export const triggersSchema = z.object({
   httpSignedData: z.array(triggerSchema),
 });
 
+export const evmAddressSchema = z.string().regex(/^0x[a-fA-F0-9]{40}$/);
+export const evmTemplateIdSchema = z.string().regex(/^0x[a-fA-F0-9]{64}$/);
+export const evmEndpointIdSchema = z.string().regex(/^0x[a-fA-F0-9]{64}$/);
```

Comment:
Really like these 3 but maybe the 2 in lines 20 and 21 can be a single hexIdSchema or something with a better name :)